.github/workflows/reusable-workflow.yml

```diff
@@ -84,14 +91,15 @@ jobs:
           - windows-2019
           - windows-2022
     env:
-      workspace_dependencies: '${{ github.workspace }}/install'
+      workspace_dependencies: 'install'
```

Comment:
I don't think it's necessary to have this. 
We can just set: `workspace_dependencies: install` 
instead of: `workspace_dependencies: ${{ env.workspace_dependencies }}` 
as we are doing in the other actions.